content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -162,7 +171,61 @@ For details, see [Create a CoreWeave AI Object Storage bucket](https://docs.core
     }
     ```
 
-The clause beginning with `"Sid": "AllowUsersInOrg"` grants users in your W&B organization direct access to the bucket. If you don't need this ability, you can omit the clause from your policy.
+    **Dedicated Cloud** / **Self-Managed**:
```

Comment:
Somewhere around here we also need to add:
In the system console user spec, add in `GORILLA_SUPPORTED_FILE_STORES: cw://` so that the server acknowledges Coreweave as a storage provider to use

Reply:
It is actually down here, step 2 in the Dedicated Cloud / Self-Managed tab: https://docs-1397.docodile.pages.dev/guides/hosting/data-security/secure-storage-connector/#team-level-byob

The section you are reading right now is what you do in the CoreWeave side, not in the W&B side. Does that make sense?

Reply:
its not the exact same thing as all we just need `GORILLA_SUPPORTED_FILE_STORES: cw://` and it has nothing to do with finding the storage address. So maybe it should be in the step 2 of storage address instead.

ie. If you have principals set up, no need to find and input storage address, just put in `GORILLA_SUPPORTED_FILE_STORES: cw://`

Reply:
Done, but I'm wondering what happens if you need CoreWeave and something else. The `:` is usually the separator for compound environment variables (thinking of `PATH=some/new/path:$PATH` to push an additional value). How would I specify CoreWeave and AWS here?